### ISSUE TITLE
epsilon in CIFAR10

### ISSUE THREAD

**Issue by sungyoon-lee** [locuslab/convex_adversarial#31]
Hi,
I've run the code like the following for epsilon=8/255=0.03137 on CIFAR-10:
python examples/cifar.py --proj 50 --norm_train l1_median --norm_test l1 --starting_epsilon 0.001 --epsilon 0.03137 --schedule_length 20 --epochs 60 --cuda_ids 1

However, I've noticed that, in the case of CIFAR, you've used the normalization with the standard deviation 0.225 in the loader
[](https://github.com/locuslab/convex_adversarial/blob/b3f532865cc02aeb6c020b449c2b1f0c13c7f236/examples/problems.py#L206)
, but epsilon is just added to the normalized input.
[](https://github.com/locuslab/convex_adversarial/blob/b3f532865cc02aeb6c020b449c2b1f0c13c7f236/convex_adversarial/dual_inputs.py#L129)
 So do I have to normalize the epsilon? like 0.03137/0.225?

FYI, with eps=0.03137 I can get the errors
 * Robust error 0.519	Error 0.392
which are much better than those reported in the paper
 * Robust error 0.792	Error 0.722
and similar to the results for eps=2/255 (Note that 2/255~0.03137/0.225)
 * Robust error 0.528	Error 0.389.

**Comment by riceric22** [comment]
Correct, the epsilon needs to be normalized. I was manually doing this in my experiments by passing in the normalized epsilon directly to the script, but I forgot the default parameter is not normalized. 

~Eric

**Comment by sungyoon-lee** [comment]
Thanks for the quick response.
